src/util/library/libraryFunctions.ml

```diff
@@ -657,6 +684,18 @@ let glibc_desc_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("getdtablesize", unknown []);
     ("daemon", unknown [drop "nochdir" []; drop "noclose" []]);
     ("putw", unknown [drop "w" []; drop "stream" [r_deep; w_deep]]);
+    (* RPC library start *)
+    ("clntudp_create", unknown [drop "addr" [r]; drop "prognum" []; drop "versnum" []; drop "wait" [r]; drop "sockp" [w]]);
+    ("clntudp_bufcreate", unknown [drop "addr" [r]; drop "prognum" []; drop "versnum" []; drop "wait" [r]; drop "sockp" [w]; drop "sendsize" []; drop "recosize" []]);
+    ("svctcp_create", unknown [drop "sock" []; drop "send_buf_size" []; drop "recv_buf_size" []]);
+    ("authunix_create_default", unknown []);
+    ("clnt_broadcast", unknown [drop "prognum" []; drop "versnum" []; drop "procnum" []; drop "inproc" [r; c]; drop "in" [w]; drop "outproc" [r; c]; drop "out" [w]; drop "eachresult" [c]]);
+    ("clnt_sperrno", unknown [drop "stat" []]);
+    ("pmap_unset", unknown [drop "prognum" []; drop "versnum" []]);
+    ("svcudp_create", unknown [drop "sock" []]);
+    ("svc_register", unknown [drop "xprt" [r_deep; w_deep]; drop "prognum" []; drop "versnum" []; drop "dispatch" [r; w; c]; drop "protocol" []]);
+    ("svc_run", special [] Abort);
```

Comment:
We should leave this as `unknown` for now and maybe in a separate PR make a new kind of special (e.g. `NoReturn`) for this. If it never returns, then the following node will be dead (like `Abort`), but the program doesn't exit (so it shouldn't be `Abort`). This makes a difference for the memLeak analysis.